src/types/ScreenInfoUiaProviderBase.cpp

```diff
@@ -9,21 +9,21 @@
 using namespace Microsoft::Console::Types;
 
 // A helper function to create a SafeArray Version of an int array of a specified length
-SAFEARRAY* BuildIntSafeArray(std::basic_string_view<int> data)
+SAFEARRAY* BuildIntSafeArray(gsl::span<const int> data)
 {
     SAFEARRAY* psa = SafeArrayCreateVector(VT_I4, 0, gsl::narrow<ULONG>(data.size()));
     if (psa != nullptr)
     {
-        for (size_t i = 0; i < data.size(); i++)

```

Comment:
Well that was obviously a past brain fart over using the foreach loop with incrementing lIndex. Sheesh.

Reply:
yup! and we only caught it because i had to make an `at` replacement lol